src/modules/graphql/schemas/student.gql

```diff
@@ -9,7 +9,7 @@ type Student {
   gender: Gender
   class: Class
   schedule: [TimeSlot!]
-  reminders: Reminders
+  reminders: [Reminder!]
```

Comment:
Shouldn't the exclamation  mark(!) be outside the square brackets? 
A student may have 0 reminders (right?) and an empty schedule (derived from class).

Reply:
Hi @doronbali, your option means that reminders field is non nullable.
I wanted to allow a nullable reminders field + enforce that any reminder item in the array is not null.
This enforcement allows the array:
a. to include only items of type reminder
b. to have 0 items (empty)